ouroboros-consensus-shelley-test/src/Test/Consensus/Shelley/Examples.hs

```diff
@@ -275,9 +276,22 @@ examples mkValidatedTx value txBody auxiliaryData = Golden.Examples {
         (mkKeyHash 0)
         (SL.emptyPParamsUpdate {SL._keyDeposit = SJust (SL.Coin 100)})
 
+mkWitnessesSAM ::
```

Comment:
What does the `SAM` stand for?

Reply:
I think it is "ShelleyAllegraMary" as it is only used in those three cases

Reply:
Yes, that's why we wrote `SAM`. We could change it to `PreAlonzo`, now that we've used that phrase in a similar workaround elsewhere.

Reply:
I pushed a commit that changed `SAM` to `PreAlonzo`.